package.json

```diff
@@ -57,7 +57,8 @@
     "@nivo/sunburst": "^0.83.0",
```

Comment:
(Ignore the location of this comment)

We need to add some quick details in the documentation before merging this update.
Nothing too crazy, just a couple paragraphs with examples on the translator page.

(While we're at it, maybe we can rename the extension to `Text2Cypher` to match the language now used by the engineering team?)